src/main/java/com/loofah/graph/api/queries/SkillsByCategoryQuery.java

```diff
@@ -0,0 +1,35 @@
+package com.loofah.graph.api.queries;
+
+import com.loofah.graph.api.models.database.Skill;
+import com.loofah.graph.api.repositories.SkillRepository;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class SkillsByCategoryQuery implements DataFetcher<List<Skill>> {
+
+    private SkillRepository skillRepository;
+
+    @Autowired
+    public SkillsByCategoryQuery(SkillRepository skillRepository) {
+        this.skillRepository = skillRepository;
+    }
+
+    @Override
+    public List<Skill> get(DataFetchingEnvironment dataFetchingEnvironment) {
+
+        String categoryId = dataFetchingEnvironment.getArgument("categoryId");
+
+        List<Skill> skills = skillRepository.findAll();
+
+        return skills.stream()
```

Comment:
Nitpick: we can make Mongo do this for us (slightly magically) by defining `List<Skill> findByCategoryId(String categoryId);` in the repository interface and then using it here. Not sure if it's any more efficient doing the filtering in the database query rather than the code?